app/models/media_object.rb

```diff
@@ -366,6 +370,33 @@ def reload
     super
   end
 
+  def self.autocomplete(query, id)
```

Comment:
I'm wondering if we'll ever want/need an autocomplete for MediaObjects themselves and might want to use this method for that.  But we don't have a use case for it right now so I think this works.  If that ever comes up then we can do the work then to take series out to it's own simple model.